apps/admin-gui/src/app/shared/perun-nav/perun-nav.component.html

```diff
@@ -52,7 +52,10 @@
     </span>
   </button>
   <mat-menu #menu="matMenu">
-    <button [routerLink]="['/myProfile']" mat-menu-item>{{'NAV.PROFILE' | translate}}</button>
+    <button *ngIf="profileUrl !== null" (click)="redirectToProfile()" mat-menu-item>
```

Comment:
If the profileUrl is not specified, there is an empty string. This way, the button will not show even if the string is empty.
```suggestion
    <button *ngIf="profileUrl" (click)="redirectToProfile()" mat-menu-item>
```